tests/core/test_proxy.py

```diff
@@ -24,7 +24,7 @@
 
 log = logging.getLogger('tests')
 
-CONTAINER_TO_RUN = "datadog/squid"
+CONTAINER_TO_RUN = "jfullaondo/squid"
```

Comment:
how did you build this image?

(a small comment explaining this would be nice :) )

Reply:
Dropped this commit, the original squid image was actually there, it was a different issue with travis. Anyways, we're off travis now, so nevermind.